src/subdataframe/subdataframe.jl

```diff
@@ -316,9 +316,12 @@ function is_column_insertion_allowed(df::AbstractDataFrame)
     throw(ArgumentError("Unsupported data frame type"))
 end
 
-function _replace_columns!(sdf::SubDataFrame, newdf::DataFrame)
-    colsmatch = _names(sdf) == _names(newdf)
-
+function _replace_columns!(sdf::SubDataFrame, newdf::DataFrame, wastransform::Bool)
```

Comment:
Maybe a name like `keep_present` would be easier to follow (i.e. say what it does rather than when it's set)? Could also make it a keyword argument, as it's hard to guess what it means when seeing just a Bool value.